docs/canvas/canvas-elements.asciidoc

```diff
@@ -1,150 +1,115 @@
 
-NOTE: Demo data is only supported on Canvas elements. Maps do not support demo data.
+* *{es} raw data* &mdash; Access your raw data in {es} without the use of aggregations. Use {es} raw data when you have low volume datasets, or to plot exact, non-aggregated values.
 
-Want to use a different element? You can delete the element by selecting it, clicking the *Element options* icon in the top right, then selecting *Delete*.
+* *Timelion* &mdash; Access your time series data using <<timelion,Timelion>> queries. To use Timelion queries, you can enter a query using the <<lucene-query,Lucene Query Syntax>>.
 
 [float]
-[[connect-element-data]]
-=== Connect the Canvas element to your data
-
-When you have finished using the demo data, connect the Canvas element to a data source.
+[[canvas-add-object]]
+==== Add a saved object
 
-NOTE: Maps do not support data sources. To change the map data, refer to <<maps, Elastic Maps>>.
+Add an object that you created outside of Canvas, then customize it to fit your display needs.
 
-. Make sure that the element is selected, then select *Data*.
+. Click *Embed object*.
 
-. Click *Change your data source*.
+. Select the object you want to add.
++
+[role="screenshot"]
+image::images/canvas-map-embed.gif[]
 
-[float]
-[[elasticsearch-sql-data-source]]
-==== Connect to {es} SQL
+. To use the customization options, click the panel menu, then select one of the following options:
 
-Access your data in {es} using SQL syntax. For information about SQL syntax, refer to {ref}/sql-spec.html[SQL language].
+* *Edit map* &mdash; Opens <<maps,Maps>> so that you can edit the original map.
 
-. Click *{es} SQL*.
+* *Customize panel* &mdash; Specifies the object title options.
 
-. In the *{es} SQL query* box, enter your query, then *Preview* it.
+* *Inspect* &mdash; Allows you to drill down into the element data.
 
-. If everything looks correct, *Save* it.
+* *Customize time range* &mdash; Exposes a time filter dedicated to the map.
 
 [float]
-[[elasticsearch-raw-doc-data-source]]
-==== Connect to {es} raw data
+[[canvas-add-image]]
+==== Add your own image
 
-Access your raw data in {es} without the use of aggregations. Use {es} raw data when you have low volume datasets, or to plot exact, non-aggregated values.
+To personalize your workpad, add your own logos and graphics.
 
-To use targeted queries, you can enter a query using the <<lucene-query,Lucene Query Syntax>>.
+. Click *Manage assets*.
 
-. Click *{es} raw documents*.
+. On the *Manage workpad assets* window, drag and drop your images.
 
-. In the *Index* field, enter the index pattern that you want to display.
-
-. From the *Fields* dropdown, select the associated fields you want to display.
-
-. To sort the data, select an option from the *Sort Field* and *Sort Order* dropdowns.
-
-. For more targeted queries, enter a *Query* using the Lucene query string syntax.
+. To add the image to the workpad, click the *Create image element* icon.
```

Comment:
Might be nice to include an image of the icon in the instruction,

Reply:
The gif shows this